app/models/chargeback_rate.rb

```diff
@@ -70,4 +72,9 @@ def self.seed
       end
     end
   end
+
```

Comment:
@tenderlove @chessbyte Well, this one will be done now, other implementations can take place going forward when other work on controller/model pairings is required.